lib/components/Dropdown.js

```diff
@@ -24,6 +25,7 @@ export default class Dropdown extends Component {
     } = this.props;
 
     const Target = customTarget;
+    const dataCyPrefix = label ? hyphenize(label) : "nui";
```

Comment:
@datt _a We could add this condition in `hyphenize` so that we don't have to handle it everywhere.